packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-embedded-content-consumer
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
changed c668719584c746aff1a8ea61fa54421cec2d8c31